packages/opencensus-core/test/test-recorder.ts

```diff
@@ -83,101 +82,133 @@ describe('Recorder', () => {
-
-          assert.strictEqual(updatedAggregationData.value, count);
-        }
-      });
-    }
-  });
-
-  describe('Add measurements to a Last Value Aggregation Data', () => {
-    for (const testCase of testCases) {
-      const lastValueData: LastValueData = {
-        type: AggregationType.LAST_VALUE,
-        tags,
-        timestamp: Date.now(),
-        value: undefined
-      };
-
-      it(`should record measurements ${testCase.description} correctly`, () => {
-        for (const value of testCase.values) {
-          const measurement: Measurement = {measure, tags, value};
-          const updatedAggregationData =
-              Recorder.addMeasurement(lastValueData, measurement) as
-              LastValueData;
-
-          assert.strictEqual(updatedAggregationData.value, value);
-        }
-      });
-    }
-  });
-
-  describe('Add measurements to a Sum Aggregation Data', () => {
-    for (const testCase of testCases) {
-      const sumData: SumData =
-          {type: AggregationType.SUM, tags, timestamp: Date.now(), value: 0};
-
-      it(`should record measurements ${testCase.description} correctly`, () => {
-        let acc = 0;
-        for (const value of testCase.values) {
-          acc += value;
-          const measurement: Measurement = {measure, tags, value};
-          const updatedAggregationData =
-              Recorder.addMeasurement(sumData, measurement) as SumData;
-
-          assert.strictEqual(updatedAggregationData.value, acc);
-        }
-      });
-    }
-  });
-
-  describe('Add measurements to a Distribution Aggregation Data', () => {
-    for (const testCase of testCases) {
-      const distributionData: DistributionData = {
-        type: AggregationType.DISTRIBUTION,
-        tags,
-        timestamp: Date.now(),
-        startTime: Date.now(),
-        count: 0,
-        sum: 0,
-        max: Number.MIN_SAFE_INTEGER,
-        min: Number.MAX_SAFE_INTEGER,
-        mean: 0,
-        stdDeviation: 0,
-        sumSquaredDeviations: 0,
-        buckets: [
-          {highBoundary: 0, lowBoundary: -Infinity, count: 0},
-          {highBoundary: 2, lowBoundary: 0, count: 0},
-          {highBoundary: 4, lowBoundary: 2, count: 0},
-          {highBoundary: 6, lowBoundary: 4, count: 0},
-          {highBoundary: Infinity, lowBoundary: 6, count: 0}
-        ]
-      };
-
-      it(`should record measurements ${testCase.description} correctly`, () => {
-        const sentValues = [];
-        for (const value of testCase.values) {
-          sentValues.push(value);
-          const measurement: Measurement = {measure, tags, value};
-          const updatedAggregationData =
-              Recorder.addMeasurement(distributionData, measurement) as
-              DistributionData;
-
-          assertDistributionData(distributionData, sentValues);
-        }
-      });
-    }
-  });
+  for (const measure of measures) {
+    describe(
+        `Add measurements of type ${measure.type} to a Count Aggregation Data`,
```

Comment:
Simplify description: `for count aggregation data of ${measure.type} values`.

Reply:
Way better, thanks again.